tasks/src/main/java/ru/sberbank/school/task07/Counter.java

```diff
@@ -1,6 +1,8 @@
 package ru.sberbank.school.task07;
 
 
+import java.io.FileNotFoundException;
+
 /**
  * Считатель различных слов в файле.
  * <p>

```

Comment:
&lt;p&gt; tag should be placed immediately before the first word, with no space after.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1637128576>*